Source/engine.cpp

```diff
@@ -2084,245 +2084,244 @@ void engine_draw_pixel(int sx, int sy)
 #endif
 }
 // 52B96C: using guessed type char gbPixelCol;
-// 52B970: using guessed type int dword_52B970;
-// 52B99C: using guessed type int dword_52B99C;
+// 52B970: using guessed type int gbRotateMap;
+// 52B99C: using guessed type int gbNotInView;
 // 69CF0C: using guessed type int gpBufEnd;
 
-void DrawLine(int x0, int y0, int x1, int y1, UCHAR col)
+void DrawLine(int x0, int y0, int x1, int y1, BYTE col)

```

Comment:
@galaxyhaxz would you mind adding a comment referencing Wu's line drawing algorithm?

Reply:
Sure! I've been really sick today so I'm probably going to break for a few days. Could you remind me>

Reply:
Get well soon

Reply:
Hope you'll have a quick recovery @galaxyhaxz.

*preparing tea and hot soup for you*